panel/layout/__init__.py

```diff
@@ -38,6 +38,7 @@
 from .float import FloatPanel  # noqa
 from .grid import GridBox, GridSpec  # noqa
 from .gridstack import GridStack  # noqa
+from .modal import Modal
```

Comment:
Should we also add it to root, so we can `pn.Modal`?

Reply:
Good question, we do it for other layouts, so for consistency the answer should be yes I suppose.